src/main/resources/python/generate_stubs.py

```
@@ -369,8 +374,68 @@ def generate_submodules(self):
+                if op_name.startswith("__"): continue
+
+                op_func = getattr(cat_obj, op_name)
+                rna = getattr(op_func, "get_rna_type", lambda: None)()
+
+                if rna:
+                    args_sig = ["override_context: Optional[Union[Dict, 'bpy.types.Context']] = None",
+                                "execution_context: Optional[str] = None",
+                                "undo: Optional[bool] = None"]
+
+                    for prop in rna.properties:
+                        if prop.identifier == "rna_type": continue
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)